### ISSUE TITLE
BenchMark: Single CNN Layer

### ISSUE THREAD

**Issue by yugkhanna** [zalandoresearch/fashion-mnist#127]
`The model is a single layer CNN (Conv2D) which takes input tensors of dimensions (image_height, image_width, image_channels) which is (28,28,1) which is the format of Fashion-MNIST dataset.

Layer 1 : Conv2D for convolution operation to extract features.

Layer 2 : MaxPooling2D to reduce dimension complexity and reduce training params

Layer 3 : BatchNormalization is used to normalize the output of the previous activation layer

Layer 4 : Dropout Layer to reduce overfitting

Flatten() is added to make the outout from 3D to 1D

Layer 5 and 6 : Dense Layers to process the output and finally pass the 10 labels for output and applying softmax activation

Results

Training Accuracy : 98.5%

Test Accuracy : 93%`

Code can be found here : "https://github.com/yugkhanna/Fashion-MNIST"

**Comment by kashif** [comment]
the numbers do not look like they are from Fashion-MNIST... on keras i would recommend you use the `dataset` API to load the data. See: https://keras.io/datasets/#fashion-mnist-database-of-fashion-articles

and then plot the results again
